### PR TITLE
Integrate ffmpeg package, export videos with audio, add Audio tag

### PR DESCRIPTION
**Pull request by justusmattern27**

### CODE REVIEW

**Review by hkonsti — APPROVED** [review]
Really cool!!

File: packages/core/src/app/Exporter.ts [hunk 1]
```diff
@@ -97,4 +102,16 @@ export interface Exporter {
...
+
+  /**
+   * Finish the rendering process.
+   *
+   * @remarks
+   * Guaranteed to be called after the rendering has finished - no matter the
+   * result. Can be used to finalize the exporting and perform any necessary
+   * clean-up.
+   *
+   * @param result - The result of the rendering.
```
[the remark this answers is not in the document]
Comment (justusmattern27) [reply packages/core/src/app/Exporter.ts]: true, removed

File: packages/core/src/scenes/Scene.ts [hunk 2]
```diff
@@ -331,6 +336,15 @@ export interface Scene<T = unknown> {
    */
   isCached(): boolean;
 
+  /**
+   * Get all media assets
+   *
+   * @remarks
```
[the remark this answers is not in the document]
Comment (justusmattern27) [reply packages/core/src/scenes/Scene.ts]: done

File: packages/core/src/app/Renderer.ts [hunk 3]
```diff
@@ -253,11 +267,21 @@ export class Renderer {
...
+
+    await this.exporter.kill?.();
+
     this.exporter = null;
+
     return result;
   }
 
   private async reloadScenes(settings: RendererSettings) {
+    console.log('reloadScenes was called');
```
[the remark this answers is not in the document]
Comment (justusmattern27) [reply packages/core/src/app/Renderer.ts]: removed it

**Comment by justusmattern27** [comment]
@hkonsti I think this is ready now. Can I merge?

**Pull request merged**